src/core/client/ui/components/TextField/TextField.css

```diff
@@ -1,8 +1,33 @@
 .root {
   composes: textField from "talk-ui/shared/typography.css";
-  background: var(--palette-common-white);
-  border: 1px solid var(--palette-grey-lighter);
+  position: relative;
+  display: block;
+  padding: calc(0.5 * var(--spacing-unit));
   box-sizing: border-box;
   border-radius: var(--round-corners);
-  padding: calc(0.5 * var(--spacing-unit));
+}
+
+.colorRegular {
+  background-color: var(--palette-common-white);
+  color: var(--palette-common-black);
+  border: 1px solid var(--palette-grey-light);
+}
+
+.colorError {
+  background-color: var(--palette-common-white);
+  border-color: var(--palette-error-main);
+  border: 2px solid var(--palette-error-darkest);
+}
+
+.fullWidth {
+  width: 100%;
+}
+
+::placeholder {
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: normal;
+  line-height: normal;
+  font-size: calc(16rem / var(--rem-base));
+  color: var(--palette-grey-lighter);
```

Comment:
Let's move this to typography

Reply:
👍

Reply:
I think it's this decision is okay but `[Error] Error: composition is only allowed when selector is single :local class name not in "::placeholder", "::placeholder" is weird` I'm getting that error.